apps/src/lib/levelbuilder/code-docs-editor/ProgrammingClassEditor.jsx

```diff
@@ -20,25 +21,19 @@ function useProgrammingClass(initialProgrammingClass) {
     setProgrammingClass({...programmingClass, [key]: value});
   }
 
-  return [programmingClass, updateProgrammingClass];
+  return [programmingClass, updateProgrammingClass, setProgrammingClass];
 }
 
 function renderExampleEditor(example, updateFunc) {
-  return (
-    <ExampleEditor
-      example={example}
-      updateExample={(key, value) => updateFunc(key, value)}
-    />
-  );
+  return <ExampleEditor example={example} updateExample={updateFunc} />;
 }
 
 function renderFieldEditor(field, updateFunc) {
-  return (
-    <FieldEditor
-      field={field}
-      updateField={(key, value) => updateFunc(key, value)}
-    />
-  );
+  return <FieldEditor field={field} updateField={updateFunc} />;
```

Comment:
Just making sure, the updates to `updateField` are b/c the previous more complicated function definition was redundant?

Reply:
Yep! I just cleaned it up while I was here